src/App.tsx

```diff
@@ -1,26 +1,143 @@
-/* eslint-disable max-len */
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { AddTodos, DeleteTodos, getTodos, USER_ID } from './api/todos';
+import { AddTodo, Todo } from './types/Todo';
+import { ErrorNotification } from './Components/ErrorNotification';
+import { Header } from './Components/Header';
+import { Main } from './Components/Main';
+import { Footer } from './Components/Footer';
+import useError from './hooks/UseError';
 
-const USER_ID = 0;
+export enum FiltredValue {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
-
-  return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">
-          React Todo App - Load Todos
-        </a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const { errorMessage, setErrorMessage } = useError();
+  const [filter, setfilter] = useState<FiltredValue>(FiltredValue.All);
+  const [allActive, setAllActive] = useState(false);
+  const [disableBtn, setDisableBtn] = useState(true);
```

Comment:
```suggestion
  const [isDisabled, setIsDisabled] = useState(true);
```
"disable", with verb sounds like a function

Reply:
Still not fixed